### PR TITLE
[dk-fortids] Finalize processing code

### PR DESCRIPTION
**Pull request by Vesihiisi**
Make sure that the preview page is ready for release:

https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/dk-fortidsminder_(da)/preview

Add reporting of unmatched values.

Task: https://phabricator.wikimedia.org/T158567

### CODE REVIEW

**Comment by Vesihiisi** [comment]
Note: Has to be adjusted to recent changes in importer and Monument.

**Comment by Vesihiisi** [comment]
The changes in master have been implemented, so technically it works.

File: importer/DkFortidsDa.py [hunk 1]
```diff
@@ -16,56 +16,86 @@ def update_descriptions(self):
...
         if self.has_non_empty_attribute("kommune"):
             if utils.count_wikilinks(self.kommune) == 1:
-                adm_location = utils.q_from_first_wikilink("da", self.kommune)
-                self.add_statement("located_adm", adm_location)
+                try:
+                    adm_q = utils.q_from_first_wikilink(
+                        "da", self.kommune)
+                    municipality = [x["item"] for x in mun_dict if x["item"] == adm_q][0]
+                    ref = self.wlm_source
+                    self.add_statement("located_adm", municipality, refs=[ref])
```
Comment (lokal-profil) [inline importer/DkFortidsDa.py]: no need to pass the ref? (also in other places)

File: importer/DkFortidsDa.py [hunk 2]
```diff
@@ -16,56 +16,86 @@ def update_descriptions(self):
...
         """
+        mun_dict = self.data_files["municipalities"]
         monument_type = utils.remove_markup(self.type).lower()
-        municipality = utils.remove_markup(self.kommune)
-        description_da = "{} i {}".format(monument_type, municipality)
+        municipality_da = utils.remove_markup(self.kommune)
+        description_da = "{} i {}".format(monument_type, municipality_da)
         self.add_description("da", description_da)
+        try:
+            location_en = [x["en"] for x in mun_dict if x["da"] == municipality_da][0]
```
Comment (Vesihiisi) [inline importer/DkFortidsDa.py]: Done.

File: importer/DkFortidsDa.py (hunk 1 above)
Comment (Vesihiisi) [inline importer/DkFortidsDa.py]: Done.

File: importer/DkFortidsDa.py [hunk 3]
```diff
@@ -16,56 +16,86 @@ def update_descriptions(self):
...
+        except IndexError:
+            location_en = "Denmark"
+        description_en = "ancient monument in {}".format(location_en)
+        self.add_description("en", description_en)
+        self.add_description("sv", "fornlämning i Danmark")
```
Comment (Vesihiisi) [inline importer/DkFortidsDa.py]: Added sv municipalities to file and descriptions.

**Pull request merged**